tree/ntuple/v7/inc/ROOT/RNTupleDescriptor.hxx

```diff
@@ -481,9 +481,8 @@ class RExtraTypeInfoDescriptor {
 private:
    /// Specifies the meaning of the extra information
    EExtraTypeInfoIds fContentId = EExtraTypeInfoIds::kInvalid;
-   /// Extra type information restricted to a certain version range of the type
-   std::uint32_t fTypeVersionFrom = 0;
-   std::uint32_t fTypeVersionTo = 0;
+   /// Type version the extra type information is bound to
+   std::uint32_t fTypeVersion = 0;
```

Comment:
Can we explicit in the commit log why the From/To is no longer needed?

Reply:
Done.